Error during pdf conversion to image interrupts the process

If the script encounters an error during the conversion of pdf pages to png it breaks the whole process without considering other pdf into the input folder

```Bash
docker run --rm -v .:/code cad/checkconsents:1.0.0 --input_folder /code/Target/Input --configfile resources/checkconsents_config.yml --working_dir /code/Target --log_level debug

2025-03-17 10:15:30 [DEBUG] [MainThread - CheckConsents] [main] configuration: {
    'intermediates': False,
    'templates': {
        'majeur': {
            'path': pathlib.PosixPath('resources/all_templates/majeur.svg'),
            'pattern':
                'génétiques d’une personne majeure réalisé pour une finalité '
                'médicale',
            'priority': 1
        },
        'majeur_tutelle': {
            'path': pathlib.PosixPath(
                'resources/all_templates/majeur_tutelle.svg'
            ),
            'pattern': 'génétiques d’une personne majeure sous tutelle',
            'priority': 1
        },
        'mineur': {
            'path': pathlib.PosixPath('resources/all_templates/mineur.svg'),
            'pattern':
                'génétiques d’une personne mineure réalisé pour une finalité '
                'médicale',
            'priority': 1
        },
        'mineur_decede': {
            'path': pathlib.PosixPath(
                'resources/all_templates/mineur_decede.svg'
            ),
            'pattern': 'génétiques d’une personne mineure décédée',
            'priority': 1
        },
        'foetus': {
            'path': pathlib.PosixPath('resources/all_templates/foetus.svg'),
            'pattern': 'génétiques d’un foetus décédé réalisé pour une finalité médicale',
            'priority': 1
        },
        'tiers_apparente': {
            'path': pathlib.PosixPath(
                'resources/all_templates/tiers_apparente.svg'
            ),
            'pattern': '(tiers apparenté)',
            'priority': 10
        }
    },
    'pages': {
        'recherche': 'Consentement pour la conservation, dans un but de recherche'
    },
    'conversion': {'density': 300, 'opt_args': None},
    'parsing_header_limit': 5
}
2025-03-17 10:15:30 [INFO] [MainThread - CheckConsents] [main] Start processing...
2025-03-17 10:15:30 [INFO] [MainThread - CheckConsents] [main] Create output directory: /code/Target/CheckConsents_20250317-101530
2025-03-17 10:15:30 [INFO] [MainThread - CheckConsents] [main] Working directory: /code/Target/CheckConsents_20250317-101530/tmpwqubbd23
2025-03-17 10:15:30 [DEBUG] [MainThread - CheckConsents] [main] pdf: /code/Target/Input/xxxx_4_patients.pdf
2025-03-17 10:15:30 [DEBUG] [MainThread - CheckConsents] [main] pdf: xxxx_4_patients.pdf
2025-03-17 10:15:30 [INFO] [MainThread - CheckConsents] [main] Convert pdf to png
convert-im6.q16: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `/code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-%d.png' @ warning/png.c/MagickPNGWarningHandler/1668.
convert-im6.q16: cache resources exhausted `/code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-%d.png' @ error/cache.c/OpenPixelCache/4095.        
convert-im6.q16: memory allocation failed `/code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-%d.png' @ error/png.c/WriteOnePNGImage/9144.
convert-im6.q16: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `/code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-%d.png' @ warning/png.c/MagickPNGWarningHandler/1668.
convert-im6.q16: No IDATs written into file `/code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-13.png' @ error/png.c/MagickPNGErrorHandler/1642.  
2025-03-17 10:15:51 [ERROR] [MainThread - CheckConsents] [main] Conversion of /code/Target/Input/xxxx_4_patients.pdf failed, command: convert -density 300 /code/Target/Input/xxxx_4_patients.pdf /code/Target/CheckConsents_20250317-101530/tmpwqubbd23/xxxx_4_patients-%d.png
2025-03-17 10:15:51 [INFO] [MainThread - CheckConsents] [main] process done

```

--- a/consentforms/checkconsents.py
+++ b/consentforms/checkconsents.py
@@ -306,15 +306,14 @@ def main(cli_args):
                 logger.info("Convert pdf to png")
                 img_file_path = Path(path.join(tmpdirname, f"{file_name}.png"))
                 try:
-                    exit_code = images.convert_pdf_2_png(Path(pf),
-                                                        DirectoryPath(tmpdirname),
-                                                        images.default_convert_output_format(file_name),
-                                                        config.conversion.density,
-                                                        config.conversion.opt_args)
+                    images.convert_pdf_2_png(Path(pf),
+                                             DirectoryPath(tmpdirname),
+                                             images.default_convert_output_format(file_name),
+                                             config.conversion.density,
+                                             config.conversion.opt_args)
                 except images.PdfConvertError as pce:
-                    #msg = f"Error during pdf conversion: {pf}"
-                    logger.error(pce.msg)
-                    errors_pdf_convert.append(pce.msg)
+                    logger.error(pce)
+                    errors_pdf_convert.append(pce)
                     continue
 
                 logger.info("detect orientation - generate corrected image")

